intellij-plugin-structure/structure-intellij/src/main/java/com/jetbrains/plugin/structure/intellij/problems/MinorWarnings.kt

```diff
@@ -179,6 +180,19 @@ open class NonexistentReleaseInUntilBuild(
   nonexistentRelease: String = ""
 ) : SuspiciousUntilBuild(untilBuild, "Version '$nonexistentRelease' does not exist")
 
+class SuspiciousReleaseVersion(
+  descriptorPath: String,
+  releaseVersion: ProductReleaseVersion,
+  pluginVersion: String
+) : InvalidDescriptorProblem(
+  descriptorPath = descriptorPath,
+  detailedMessage = "The <release-version> parameter [$releaseVersion] and the plugin version [$pluginVersion] " +
+    "should have a matching beginning. " +
+    "For example, release version '20201' should match plugin version 2020.1.1"
+) {
+  override val level
+    get() = Level.WARNING
```

Comment:
We don't allow upload with such problem. Probably it should be error for new, warning for existing ones and ignored for JetBrains (since JetBrains plugins have very strange release versions like `2024200` instead of `20242`)